bin/node-template/pallets/template/src/lib.rs

```diff
@@ -14,8 +14,20 @@ mod tests;
 #[cfg(feature = "runtime-benchmarks")]
 mod benchmarking;
 pub mod weights;
+
+mod storages;
+mod events;
+mod errors;
+mod calls;
+
 pub use weights::*;
 
+use frame_support::pallet_macros::*;
+
+#[import_section(events)]
+#[import_section(errors)]
+#[import_section(calls)]
+#[import_section(storages)]
```

Comment:
now that I'm seeing several imported at once, would be cool to have an `#[import_sections(section1, section2, section3)]` macro that expands to the above for scenarios like this. Doesn't have to be addressed in the initial PR but making a note of it

Reply:
Yes and a comment please about what this is doing.